src/styles/search.css

```diff
@@ -0,0 +1,24 @@
+.search-input {
+  height: 100%;
+  font-size: 20px;
+  padding: 0 10px;
+  border-radius: 11px;
+  text-align: start;
```

Comment:
text-align: start is not a thing so you don't need to apply anything. Text align is for when you want the text to go in the middle or to the right see here for more info https://www.w3schools.com/cssref/pr_text_text-align.ASP